tox-envlist and virtualenv are not needed to run tests.
tox version pin is too strict.

--- domdf_python_tools-3.10.0/tox.ini.orig
+++ domdf_python_tools-3.10.0/tox.ini
@@ -39,10 +39,6 @@
     build
 skip_missing_interpreters = True
 isolated_build = True
-requires =
-    tox-envlist>=0.2.1
-    tox~=3.0
-    virtualenv!=20.16.0
 
 [envlists]
 test =
